servicetalk-http-router-jersey3-jakarta10/build.gradle

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright © 2019, 2021, 2024 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: "io.servicetalk.servicetalk-gradle-plugin-internal-library"
+
+checkstyleMain {
+  enabled false
+}
+
+checkstyleTest {
+  enabled false
+}
+
+checkstyleTestFixtures {
+  enabled false
+}
+
+spotbugsMain {
+  enabled false
+}
+
+task cleanSources(type: Delete) {
+  // Don't delete src/testFixtures/java/.gitkeep
+  // gitkeep used to ensure that a src/textFixtures/java exists or Gradle processing will fail
+  delete 'src/main', 'src/test', 'src/testFixtures/java/io'
+}
+
+task copySourcesForJersey3(type: Copy) {
+  dependsOn tasks.cleanSources
+  from '../servicetalk-http-router-jersey/src'
+  into 'src'
+  filter { line -> line.replaceAll('javax.ws.rs', 'jakarta.ws.rs') }
+  filter { line -> line.replaceAll('javax.inject', 'jakarta.inject') }
+
+  // Only modify specific javax.annotations.  Some like javax.annotation.Nullable need to stay the same
+  filter { line -> line.replaceAll('javax.annotation.Priority', 'jakarta.annotation.Priority') }
+}
+
+tasks.processResources.dependsOn(copySourcesForJersey3)
+tasks.sourcesJar.dependsOn(copySourcesForJersey3)
+tasks.compileJava.dependsOn(copySourcesForJersey3)
+tasks.processTestResources.dependsOn(copySourcesForJersey3)
+tasks.compileTestJava.dependsOn(copySourcesForJersey3)
+
+def actualJerseyVersion = "${jersey3VersionEE10}"
```

Comment:
```
/home/runner/work/servicetalk/servicetalk/servicetalk-http-router-jersey3-jakarta10-internal/src/main/java/io/servicetalk/http/router/jersey/internal/SourceWrappers.java:23: error: cannot access org.glassfish.jersey.internal.PropertiesDelegate

import org.glassfish.jersey.internal.PropertiesDelegate;
                                    ^
  bad class file: /home/runner/.gradle/caches/modules-2/files-2.1/org.glassfish.jersey.core/jersey-common/3.1.6/56f5e4e0f48a913fda0e532fe07cc7c08c0f1350/jersey-common-3.1.6.jar(org/glassfish/jersey/internal/PropertiesDelegate.class)
    class file has wrong version 55.0, should be 52.0
    Please remove or make sure it appears in the correct subdirectory of the classpath.
> Task :servicetalk-http-router-jersey3-jakarta10-internal:compileJava FAILED
```

Looks like Jersey 3.1.x requires JDK11 (3.0.x still fine with JDK8). Have a look at `servicetalk-concurrent-jdkflow/build.gradle` to see an example of how we can add a module what requires JDK11+

Reply:
done